### ISSUE TITLE
WebStorm errors, but it doesn't affect productivity

### ISSUE THREAD

**Issue by legion-zver** [beansoft/react-native-console#54]
**React Native Console 2020.3.2 EAP installed.**

```
During querying provider com.github.beansoftapp.reatnative.idea.editor.JsFilePreviewImageLineMarkerProvider@17c2f48 (class com.github.beansoftapp.reatnative.idea.editor.JsFilePreviewImageLineMarkerProvider)

java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at com.github.beansoftapp.reatnative.idea.editor.JsFilePreviewImageLineMarkerProvider.getLineMarkerInfo(Unknown Source)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.queryProviders(LineMarkersPass.java:163)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.lambda$doCollectInformation$3(LineMarkersPass.java:88)
	at com.intellij.codeInsight.daemon.impl.Divider.divideInsideAndOutsideInOneRoot(Divider.java:81)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectInformation(LineMarkersPass.java:83)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:54)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:399)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1110)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:392)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:629)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:581)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:391)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:367)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:170)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:182)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:365)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:181)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Comment by beansoft** [comment]
Thx so much for your report, I'll take a look at. Your feedback make the plugin better.

**Comment by beansoft** [comment]
Hi @legion-zver I've fixed it in https://github.com/beansoft/react-native-console/issues/53, but forgot to publish it, you can download and install it locally by this link:
https://plugins.jetbrains.com/plugin/download?rel=true&updateId=99772 

This update is pending JetBrains' approval, will be published in 2 work days.

**Comment by beansoft** [comment]
This update has been released, please update to React Native Console 2020.3.3 EAP to check it.
